core/src/runtime/raw.rs

```diff
@@ -17,6 +18,94 @@ use crate::{
 
 use super::{opaque::Opaque, InterruptHandler};
 
+const DUMP_BYTECODE_FINAL: u64 = 0x01;
+const DUMP_BYTECODE_PASS2: u64 = 0x02;
+const DUMP_BYTECODE_PASS1: u64 = 0x04;
+const DUMP_BYTECODE_HEX: u64 = 0x10;
+const DUMP_BYTECODE_PC2LINE: u64 = 0x20;
+const DUMP_BYTECODE_STACK: u64 = 0x40;
+const DUMP_BYTECODE_STEP: u64 = 0x80;
+const DUMP_READ_OBJECT: u64 = 0x100;
+const DUMP_FREE: u64 = 0x200;
+const DUMP_GC: u64 = 0x400;
+const DUMP_GC_FREE: u64 = 0x800;
+const DUMP_MODULE_RESOLVE: u64 = 0x1000;
+const DUMP_PROMISE: u64 = 0x2000;
+const DUMP_LEAKS: u64 = 0x4000;
+const DUMP_ATOM_LEAKS: u64 = 0x8000;
+const DUMP_MEM: u64 = 0x10000;
+const DUMP_OBJECTS: u64 = 0x20000;
+const DUMP_ATOMS: u64 = 0x40000;
+const DUMP_SHAPES: u64 = 0x80000;
```

Comment:
These should probably be a bitflags crate like enum. Can be done after the PR is merged.

Reply:
Bitflags crate seems like a lot of complexity for very little value. I could use a regular rust enum but that would require a lot of casts when building the flags uint.